CRM/Birthdays/BirthdayContacts.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/*
+ * Copyright (C) 2022 SYSTOPIA GmbH
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation in version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class CRM_Birthdays_BirthdayContacts
+{
+    private int $group_id;
+
+    /**
+     * @throws API_Exception
+     */
+    public function __construct()
+    {
+        $this->group_id = $this->get_group_id_from_api();
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function get_birthday_contacts_of_today($is_debug_email = ''): array
+    {
+        try {
+            if (!empty($is_debug_email)) {
+                $limit = 'LIMIT 10';
+                $day_filter = 'AND birth_date IS NOT NULL'; // just show up to 10 contacts no matter which birthdate
+            } else {
+                $limit = '';
```

Comment:
Are we sure we don't care for potentially huge numbers of birthday boys and girls causing timeouts?